### PR TITLE
Added Command Prompt shortcuts

### PR DESCRIPTION
**Pull request by grow-xd**
Added shortcuts of mark mode too

### Checklist:
- [x] Read our [contributing guidelines](../docs/CONTRIBUTING.md).
- [x] Search for duplicates.
- [x] Used an informative name for this pull request.

### Follow-up
- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: CheatSheets/windows-cheatsheet.md [hunk 1]
```diff
@@ -16,6 +16,7 @@
   - [Settings Shortcuts](#settings-shortcuts)
   - [Touchpad Shortcuts](#touchpad-shortcuts)
   - [Take Screenshot Shortcut](#take-screenshot-shortcut)
+  - [Command Prompt Shortcuts](command-prompt-shortcuts)
```
Comment (crescentpartha) [inline CheatSheets/windows-cheatsheet.md:19]:
```suggestion
  - [Command Prompt Shortcuts](#command-prompt-shortcuts)
```

**Review by crescentpartha — CHANGES_REQUESTED** [review]
The link doesn't work, add the `#`.

**Comment by grow-xd** [comment]
Done!

**Review by crescentpartha — APPROVED** [review]
Approved.

**Pull request merged**